docs/style.css

```diff
@@ -0,0 +1,557 @@
+    width: 480px;
+    height: 480px;
+    object-fit: contain;
+}
+
+.social {
+    display: inline-grid;
+    grid-auto-flow: row;
+    max-height: 100px;
+    max-width: 204px;
+    max-width: fit-content;
+    gap: 12px;
+}
+
+.row-slider {
+    display: grid;
+    grid-auto-flow: column;
+    max-width: 1152px;
+    overflow:scroll;
+    grid-template-columns: 1fr;
+    align-self: center;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+}
+
+.slider.container {
+    display: flex;
+    flex-direction: row;
+    max-width: 1360px;
+    width: 100%;
+    height: 640px;
+    align-items: center;
+    justify-content:space-between ;
+    gap: 40px;
+}
+
+.slider-button {
+    border: none;
+    width: 59px;
+    height: 59px;
+    background-color: var(--light);
+}
+
+.slider-button:hover {
+    border-radius: 50%;
+}
+
+.right-btn svg:hover  {
+    stroke: var(--light);
+    fill: var(--container);
+}
+
+.right-btn svg {
+    fill: var(--light);
+    stroke: #665F55;
+}
+
+.left-btn svg:hover  {
+    stroke: var(--light);
+    fill: var(--container);
+}
+
+.left-btn svg {
+    fill: var(--light);
+    stroke: #665F55;
+}
+
+.appstore-btn, .playstore-btn{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid var(--container);
+    gap: 8px;
+    border-radius: 100px;
+    width: 200px;
+    height: 64px;
+    padding: 12px 20px;
+    background: var(--light);
+}
+
+.playstore-btn svg:hover {
+    fill: var(--light);
+}
+
+.appstore-btn svg:hover {
+    fill: var(--light);
+}
+
+.apple-store svg:hover {
+    stroke: var(--light);
+}
+
+.playstore-btn svg {
+    fill: var(--dark);
+}
+
+.appstore-btn svg {
+    fill: var(--dark);
+    stroke: var(--dark);
+}
+
+.appstore-btn:hover, .playstore-btn:hover {
+    background: var(--container);
+}
+
+.images {
+    display: flex;
+    gap: 40px;
+}
+
+.column1, .column2 {
+    display: grid;
+    gap: 40px;
+}
+
+.about-picture {
+    border-radius: 20px;
+}
+
+.about-picture.picture-1 {
+    width: 660px;
+    height: 590px;
+}
+
+.about-picture.picture-2 {
+    width: 660px;
+    height: 430px;
+}
+
+.about-picture.picture-3 {
+    width: 100%;
+    height: 430px;
+}
+
+.about-picture.picture-4 {
+    width: 660px;
+    height: 590px;
+}
+
+.mobile-app.container {
+    display: flex;
+    gap: 100px;
+}
+
+.menu {
+    display: flex;
+    flex-direction: column;
+    margin-top: -10px;
+}
+
+.menu.container {
+    height: 1328px;
+    text-align: center;
+}
+
+.menu-offer {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    padding-bottom: 0px;
+}
+
+.menu-grid {
+    margin-top: 30px;
+}
+
+.name-and-the-price-of-the-coffee {
+    padding-left: 15px ;
+}
+
+.coffee-description {
+    margin-top: -15px;
+    padding: 0 10px 0 15px;
+}
+
+.tab-list {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 16px;
+    justify-content: space-between;
+}
+
+.item{
```

Comment:
Did you use prettier for css? Would be great to have the same spacing  and formatting everywhere:
".classname {" or ".classname{"

Reply:
Not using prettier specifically for css. It formats HTML but not the css. I'd like know the way to configure it.

Reply:
Check this SO answer, maybe it helps:
https://stackoverflow.com/questions/54092631/vs-code-how-to-auto-format-css